packages/product-react/stories/TextArea.story.tsx

```diff
@@ -0,0 +1,59 @@
+import { storiesOf } from '@storybook/react';
+import React from 'react';
+import { TextArea } from '../src/components/TextArea';
+
+storiesOf('TextArea', module)
+  .add('default', () => (
+    <TextArea
+      id="example"
+      value={undefined}
+      placeholder="hello"
+      label="im a label"
+    />
+  ))
+  .add('required', () => (
+    <TextArea
+      id="example"
+      value={undefined}
```

Comment:
passing an undefined value seems redundant, but it's just a story so i dont feel strongly about removing it

Reply:
ya true